### ISSUE TITLE
关于训练及RoboTwin迁移后评测的问题，请指教解答

### ISSUE THREAD

**Issue by LNC-22** [H-EmbodVis/TurboVLA#16]
Dear author and co-authors,

    我在复现这个结果的时候，用的4卡3090，batch8，numworkers0，55000步的时候效果还是很差，成功率几乎是0（50个任务的完成率都这样），这是因为完全没收敛造成的吗？在这种卡的情况下或许要训300000万步才能达到你们55000的类似效果吗？还是因为false掉persistent_workers和pin_memory导致的呢？随后附上config及55000步时的输出以供参考。
```yaml
datasets:
  vla_data:
    dataset_py: lerobot_datasets
    include_state: true
    data_root_dir: ${oc.env:ROBOTWIN_DATA_ROOT}
    data_mix: robotwin_clean_50
    action_type: abs_qpos
    action_mode: abs
    sequential_step_sampling: false
    per_device_batch_size: 8
    load_all_data_for_training: true
    obs_image_size: [224, 224]
    video_backend: pyav
    num_workers: 0
    prefetch_factor: 4
    persistent_workers: false
    pin_memory: false
```
`100%|██████████| 55000/55000 [37:14:35<00:00,  2.44s/it, data_times=0.688, model_times=1.737]09/16 [16:31:28] INFO     | >> Step 55000, Loss: {'action_dit_loss': 0.008227644488215446, 'mse_score':    train_starvla.py:313                                                            0.0007320002998624529, 'data_time': 0.6878268720174674, 'model_time':                                                          1.7372512349975295, 'learning_rate': 5e-05, 'epoch': 3.2})  `
    另，因RoboTwin迁移到XPolicyLab了，评测时是在XPolicyLab内新写的脚本完成评测的，这是否也会造成评测失败，而不只是模型训练的问题吗？也是这个原因（模型接口为RoboTwin2.0）造成无法在XPolicyLab评测开源的权重模型吗？

    期待得到作者们的解答。

Yours sincerely
